fpga/src/main/scala/arty/Configs.scala

```diff
@@ -54,4 +72,9 @@ class WithArtyTweaks extends Config(
 class TinyRocketArtyConfig extends Config(
   new WithArtyTweaks ++
   new chipyard.TinyRocketConfig)
+
+class ArtyWithGPIOConfig extends Config(
+  new WithArtyGPIO ++
```

Comment:
Lets avoid the `WithArtyGPIO` config above.

Add `WithArtyGPIOHarnessBinder` and `WithGPIOPassThrough` to `WithArtyTweaks`. 
Then in `ArtyWithGPIOConfig`, have
```
new chipyard.config.WithGPIOWidth(32) ++
new chipyard.config.WithGPIOIncludeIOF(true) ++
new TinyRocketArtyConfig
```

The purpose of this is to orthogonalize digital-level changes from IO-level changes from harness-level changes.

Reply:
Moving `WithArtyGPIOHarnessBinder ` and `WithGPIOPassThrough` to WithArtyTweaks, without requiring the GPIO width to be 32 and IOFs to be included, will cause the TinyRocketArtyConfig to fail generation. I think the most reasonable approach would be to include the GPIO in TinyRocketArtyConfig and then remove the new "ArtyWithGPIOConfig".